spec/jobs/google/search_keyword_job_spec.rb

```diff
@@ -0,0 +1,119 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Google::SearchKeywordJob, type: :job do
+  include ActiveJob::TestHelper
+
+  describe '#perform' do
+    context 'given a valid request' do
+      it 'saves 3 as ads_top_count in the Database' do
```

Comment:
I think we can reduce this phrase, because it's quite obvious. 🎨 

```suggestion
      it 'saves 3 as ads_top_count' do
```